flexget/components/trakt/trakt_lookup.py

```diff
@@ -194,24 +184,97 @@ class PluginTraktLookup:
         'trakt_language': 'language',
         'trakt_genres': lambda i: [db_genre.name for db_genre in i.genres],
         'trakt_languages': 'translation_languages',
-    }
-
-    movie_translate_map = {
+    },
+    'movie_actors': {'trakt_actors': lambda movie: db.list_actors(movie.actors)},
+    'movie_translations': {
         'trakt_translations': lambda movie: db.get_translations_dict(movie.translations, 'movie')
-    }
+    },
+}
+
+
+@entry.register_lazy_lookup('trakt_lazy_lookup')
+def lazy_lookup(entry, lazy_lookup_name, media_type):
+    with Session() as session:
+        try:
+            db_data = get_db_data_for(media_type, entry, session)
+        except LookupError as e:
+            logger.debug(e)
+        else:
+            entry.update_using_map(lazy_lookup_types[lazy_lookup_name], db_data)
+    return entry
+
+
+def add_lazy_fields(entry: entry.Entry, lazy_lookup_name, media_type):
```

Comment:
should we use type hinting for all new / modified code?

Reply:
Yeah, I think it's a pretty good idea. Definitely for functions that might be called from outside the module.